mmseg/models/backbones/beit.py

```diff
@@ -61,9 +61,14 @@ def __init__(self,
         if qv_bias:
             self.q_bias = nn.Parameter(torch.zeros(embed_dims))
             self.v_bias = nn.Parameter(torch.zeros(embed_dims))
+            self.qkv_bias = torch.cat(
```

Comment:
May cause the problem that loading the old ckpt doesn't affect self.qkv_bias. mIoU will be affected